ngraph/core/reference/src/runtime/reference/convert.cpp

```diff
@@ -0,0 +1,273 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/runtime/reference/convert.hpp"
+#include "jit_generator.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            namespace
+            {
+                class jit_convert_u8_to_f16 : public jit::Generator
```

Comment:
`jit_convert_u8_to_f16` and `jit_convert_f16_to_f32` are mostly copy-paste each other. Can we have one jit_precision_convert generator passing src/dst precisions as ctor arguments?

Reply:
fixed